<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>clientvklad</class>
 <widget class="QDialog" name="clientvklad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>940</width>
    <height>408</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Вклады клиента</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0" rowspan="2">
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="3" column="0">
       <widget class="QLabel" name="lbPart">
        <property name="text">
         <string>Погашенная часть</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="2">
       <widget class="QLineEdit" name="lePart"/>
      </item>
      <item row="2" column="1" colspan="2">
       <widget class="QLineEdit" name="leDateClos"/>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="leDateOpen"/>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lbDateClos">
        <property name="text">
         <string>Дата закрытия</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1">
       <widget class="QPushButton" name="btnEdit">
        <property name="text">
         <string>Изменить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="2">
       <widget class="QLineEdit" name="leID"/>
      </item>
      <item row="4" column="2">
       <widget class="QPushButton" name="btnDel">
        <property name="text">
         <string>Удалить</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lbID">
        <property name="text">
         <string>Вклад</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QPushButton" name="btnAdd">
        <property name="text">
         <string>Добавить</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lbDateOpen">
        <property name="text">
         <string>Дата открытия</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QPushButton" name="btnConnect">
     <property name="text">
      <string>Подключение</string>
     </property>
    </widget>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="btnSearch">
     <property name="text">
      <string>Поиск</string>
     </property>
    </widget>
   </item>
   <item row="0" column="3">
    <widget class="QLineEdit" name="leSearch"/>
   </item>
   <item row="0" column="4">
    <widget class="QPushButton" name="btnSelectAll">
     <property name="text">
      <string>Выбрать Все</string>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="btnExit">
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QTableWidget" name="twSec"/>
   </item>
   <item row="1" column="1" rowspan="3" colspan="4">
    <widget class="QTableWidget" name="twClientVklad"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
